### ISSUE TITLE
Transkriptionen: Add arrow and keyboard navigation

### ISSUE THREAD

**Issue by domsteinbach** [DHBern/presentation_parzival#275]
The project would like to navigate via arrow keys and arrow buttons from one page (Steite/Blatt) to the next/previous.

As multiple Textzeugen can be displayed the keyboard navigation only works if only one Textzeuge is displayed. (navigating both makes them "async" and results in very ugly stacking snackbars if one Handschrift does not have a next page. ) .

The keyboard navigation is also disabled when inputs are focused.
